docs/en/04_Changelogs/4.7.0.md

```diff
@@ -0,0 +1,48 @@
+# 4.7.0 (Unreleased)
+
+## Overview
+
+- [Experimental support for PHP 8](#experimental-support-for-php-8)
+- [Support for Symfony 4 Components](#support-for-symfony-4-components)
+- [Default MySQL collation updated](#default-mysql-collation-updated)
+
+## New features
+
+### Experimental support for PHP 8
+
+You can now run Silverstripe CMS on PHP 8, which is due for release in November 2020. PHP 8 includes
+many exciting new features like named parameters, attributes, and union types.
+
+Upgrading to PHP 8 on projects using `SapphireTest` for unit testing requires adoption of the
+[`sminnee/phpunit`](https://packagist.org/packages/sminnee/phpunit) fork, and Prophecy is not
+currently supported. A future release will adopt the current mainline version of PHPUnit.
```

Comment:
"We hope to give you better support for choosing your own testing tools, including never versions of PHPUnit, in a future release."